doc/hardware.rst

```diff
@@ -0,0 +1,426 @@
+    you can select other pins and pass :py:attr:`gpio_DC` and/or :py:attr:`gpio_RST`
+    argument specifying the new *GPIO* pin numbers in your serial interface create
+    call (this applies to PCD8544, ST7567 and ST7735).
+
+  * Because CE is connected to CE0, the display is available on SPI port 0. You
+    can connect it to CE1 to have it available on port 1. If so, pass
+    :py:attr:`port=1` in your serial interface create call.
+
+The GPIO pins used for this SPI connection are the same for all versions of the
+Raspberry Pi, up to and including the Raspberry Pi 3 B.
+
+========== ====== ============ ======== ==============
+Device Pin Name   Remarks      RPi Pin  RPi Function
+========== ====== ============ ======== ==============
+1          VCC    +3.3V Power  P01-17   3V3
+2          GND    Ground       P01-20   GND
+3          D0     Clock        P01-23   GPIO 11 (SCLK)
+4          D1     MOSI         P01-19   GPIO 10 (MOSI)
+5          RST    Reset        P01-22   GPIO 25
+6          DC     Data/Command P01-18   GPIO 24
+7          CS     Chip Select  P01-24   GPIO 8 (CE0)
+========== ====== ============ ======== ==============
+
+.. note::
+
+  * When using the 4-wire SPI connection, Data/Command is an "out of band" signal
+    that tells the controller if you're sending commands or display data. This
+    line is not a part of SPI and the library controls it with a separate GPIO
+    pin. With 3-wire SPI and I2C, the Data/Command signal is sent "in band".
+
+  * If you're already using the listed GPIO pins for Data/Command and/or Reset,
+    you can select other pins and pass a ``gpio_DC`` and/or a ``gpio_RST``
+    argument specifying the new *BCM* pin numbers in your serial interface create
+    call.
+
+  * The use of the terms 4-wire and 3-wire SPI are a bit confusing because, in
+    most SPI documentation, the terms are used to describe the regular 4-wire
+    configuration of SPI and a 3-wire mode where the input and output lines, MOSI
+    and MISO, have been combined into a single line called SISO. However, in the
+    context of these LCD controllers, 4-wire means MOSI + Data/Command and 3-wire
+    means Data/Command sent as an extra bit over MOSI.
+
+  * Because CS is connected to CE0, the display is available on SPI port 0. You
+    can connect it to CE1 to have it available on port 1. If so, pass
+    ``port=1`` in your serial interface create call.
+
+Enable the SPI port::
+
+    $ sudo raspi-config
+    > Advanced Options > A6 SPI
+
+If ``raspi-config`` is not available, enabling the SPI port can be done
+`manually <http://elinux.org/RPiconfig#Device_Tree>`_.
+
+Ensure that the SPI kernel driver is enabled::
+
+  $ ls -l /dev/spi*
+  crw-rw---- 1 root spi 153, 0 Nov 25 08:32 /dev/spidev0.0
+  crw-rw---- 1 root spi 153, 1 Nov 25 08:32 /dev/spidev0.1
+
+or::
+
+  $ lsmod | grep spi
+  spi_bcm2835             6678  0
+
+Then add your user to the *spi* and *gpio* groups::
```

Comment:
doubleback ticks for spi and gpio

Reply:
Consistency issue as above.

Reply:
i dont mind consistency doc issues between core and sub-projects. At some point it'll be fixed in both..